src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTest.cs

```diff
@@ -172,41 +185,32 @@ private void ReportDifferences()
     {
         if (Differences.Length > 0)
         {
-            
             if (Config.WarnOnSdkContentDiffs)
             {
                 OutputHelper.LogWarningMessage(Differences.ToString());
             }
             else
             {
                 OutputHelper.WriteLine(Differences.ToString());
```

Comment:
When I looked at the pipeline run, the differences were not outputted to the Azdo console. Is this expected behavior?

Reply:
Yes, you must look at the test view to see the output - https://dev.azure.com/dnceng/internal/_build/results?buildId=2391755&view=ms.vss-test-web.build-test-results-tab&runId=53163663&resultId=100002&paneView=attachments.

In general this is the best view whenever there is any type of test failure.